src/App/Header/UserMenu/Menu.tsx

```diff
@@ -1,40 +1,56 @@
-import { WithAuthenticatorProps } from "@aws-amplify/ui-react";
 import { Link } from "react-router-dom";
+import { AuthenticatedUser } from "slices/auth";
 import { appRoutes } from "constants/routes";
 
-type Props = Required<WithAuthenticatorProps> & {
+type Props = {
   classes?: string;
-  isLoading: boolean;
+  signOut(): void;
+  user: AuthenticatedUser;
 };
-
-export default function Menu({
-  classes = "",
-  isLoading,
-  user,
-  signOut,
-}: Props) {
+export default function Menu({ classes = "", user, signOut }: Props) {
   return (
     <div
       className={`${classes} bg-white dark:bg-blue-d6 w-max rounded overflow-hidden`}
     >
       <p className="text-sm p-2 text-gray-d1 dark:text-gray">
-        {user?.attributes?.given_name
-          ? `Welcome, ${user.attributes.given_name}!`
-          : user?.attributes?.email
-          ? `Welcome, ${user.attributes.email}!`
-          : "Welcome!"}
+        Welcome, {user.firstName || user.email}
       </p>
-      <Link to={appRoutes.donations}>My Donations</Link>
-      {user.attributes?.endowments && (
-        <Link to={`${appRoutes.admin}/${user.attributes.endowments}`}>
+
+      {/**
+       * TODO: temp hidden until donations/:user-id is protected to viewed only
+       * by authenticated user with user-id
+       */}
+      <div className="hidden p-4 border-t border-prim">
+        <Link
+          to={appRoutes.donations}
+          className="text-orange hover:text-orange-l2 text-sm uppercase"
+        >
+          My Donations
+        </Link>
+      </div>
+
+      {/** TODO: temp hidden until user endowments are in jwt claim  */}
```

Comment:
@SovereignAndrey, temp set this to `hidden` until proper authorization is attached in the authenticated user